### PR TITLE
Add additional questions to release command

### PR DESCRIPTION
**Pull request by niklasnatter**

### CODE REVIEW

File: bin/release [hunk 1]
```diff
@@ -111,31 +111,51 @@ function pushTag(SymfonyStyle $ui, string $directory, string $branch, $tag): voi
 
         $ui->section('Configuration');
 
-        if ('y' !== \strtolower($ui->ask(\sprintf('Create tags for branch "%s"?', $branch), 'y'))) {
+        if ('y' !== \strtolower($ui->ask(\sprintf('Create new release for branch "%s"?', $branch), 'y'))) {
```
Comment (alexander-schranz) [inline bin/release]: actually it create a tag not a release

File: bin/release [hunk 2]
```diff
@@ -111,31 +111,51 @@ function pushTag(SymfonyStyle $ui, string $directory, string $branch, $tag): voi
...
             $ui->text('Canceled by user.');
 
             return 0;
         }
 
-        $ui->text(\sprintf('Start tagging "%s" branch.', $branch));
+        $ui->text(\sprintf('Start creating release for "%s" branch.', $branch));
```
Comment (alexander-schranz) [inline bin/release]: same as above it create a tag not a release

File: bin/release (hunk 1 above)
Comment (niklasnatter) [reply bin/release]:
```suggestion
        if ('y' !== \strtolower($ui->ask(\sprintf('Create tags for branch "%s"?', $branch), 'y'))) {
```

File: bin/release (hunk 2 above)
Comment (niklasnatter) [reply bin/release]:
```suggestion
        $ui->text(\sprintf('Start tagging "%s" branch.', $branch));
```

Comment (niklasnatter) [reply bin/release]: reverted this change 🙂

**Review by alexander-schranz — APPROVED** [review]

**Pull request merged**